benchmarks/src/main/terraform/aws/modules/network/main.tf

```diff
@@ -0,0 +1,87 @@
+##
```

Comment:
Can we re-use at least some module of the existing templates in the templates module?

Reply:
Yes, definitely. I would like to do that as much as possible. Right now, I can't estimate what could be shared and what couldn't. My plan is to finish this and make it working  separately and identify and reuse shared parts in a subsequent PR. IMHO, templates needs to be refactored a bit to be more generic and I don't want to everything at once and end up with one big bang PR. How does it sound to you?

Reply:
ok